### ISSUE TITLE
[proposal] Timezone

### ISSUE THREAD

**Issue by Rizzrack09** [epochtalk/epochtalk#760]
Timezone takes browser time into account as default.

Let's assume my local time in GMT+4. 
My browser time is GTM+2 (I use Tor, VPN, etc)
So I select +2 in the offset. The function seems to add 2 hours to GMT ... so GTM+2 ...so nothing changes.
If I select +3 in the offset, it adds 1 hour (GMT + 2 + 1)
To get to GMT+4 (as I want) I actually have to select +4 in the offset, even tho difference is just 2h

Example:
Current time where I am is 8PM (GMT+4)
Browser time is 6PM (GMT + 2)
GMT time is 4PM
offset +1 = 4PM + 1h = 5PM
offset + 2 = 4PM + 2h = 6PM
offset + 3 = 4PM + 3h = 7PM
offset + 4 = 4PM + 4h = 8PM

It does work but it's very confusing at first
Detected timezone should be displayed. Something like the rough mockup below:
![timezone_bug](https://user-images.githubusercontent.com/51131936/73870901-82142600-4855-11ea-8bb5-b8802a18c864.JPG)
So you understand if you are from New York for example you have to select -5 (GTM-5) and after daylight savings time they have to manually adjust that. No problem :)
Tho with this Guest users will see timestamps depending on their browser time, which might be different from browser to browser. However seems to be the same thing as having a general forum time. For some users there will always be differences if they don't select offset.

Great job on the timezone function !
After all it seems just the text above the drop-down should be slightly changed

**Comment by unenglishable** [comment]
@Rizzrack09 yes, this feature was intended to be a UTC/GMT offset, and does not take into account what the user's browser time is.  thanks for letting us know it's confusing - i'll update the text to match what the feature is doing 👍
